docs/rafter/10-01-rafter-troubleshooting.md

```diff
@@ -0,0 +1,22 @@
+---
+title: AssetGroup restore fails due to duplicated default buckets
+type: Troubleshooting
+---
+
+[Velero restores Kyma resources asynchronously](/root/kyma/#tutorials-restore-with-velero), following the alphabetical `kind` object order. This creates issues for Rafter as Velero first restores (Cluster)AssetGroups and then buckets. When the (Cluster)AssetGroup controller notices there are no default buckets available, it automatically tries to create them. If Velero restores the default buckets in the meantime, (Cluster)AssetGroups restoration will fail due to too many buckets with the `rafter.kyma-project.io/access:public` label.
```

Comment:
```suggestion
It may happen that (Cluster)AssetGroups cannot be proceeded due to too many default buckets in the cluster with the `rafter.kyma-project.io/access:public` label.
```